src/main/reactjs/public/i18n/fi-FI/translation.json

```diff
@@ -341,7 +341,7 @@
           "title": "Hae auktorisoitua kääntäjää"
         },
         "noSearchResults": "Ei hakutuloksia",
-        "note": "Huomaa, että kielittäin haettaessa mahdolliset hakuvaihtoehdot ovat kotimaisista kielistä (suomi, ruotsi, saamen kielet) vieraisiin kieliin tai toisinpäin sekä kotimaisten kielten välillä.",
+        "note": "Kielittäin haettaessa mahdolliset hakuvaihtoehdot ovat kotimaisista kielistä (suomi, ruotsi, saamen kielet) vieraisiin kieliin tai toisinpäin sekä kotimaisten kielten välillä.",
```

Comment:
Figman vastine taitaa olla vanhentunut, jossa tämä alku muotoa `Huomaa, että kieliparilla haettaessa...` ? Mietin muuten, että tämä huomio voisi olla esimerkiksi `Kielittäin haettaessa vähintään toisen hakuvaihtoehdon on oltava kotimainen kieli (suomi, ruotsi, saamen kielet).`. Toki OPH nyt päättää viime kädessä mitä tälle tehdään.